etas/oef/data/europe_parameters.json

```diff
@@ -0,0 +1,89 @@
, array([-22.93258091,  62.6242536 ]), array([-22.93258091,  62.6242536 ]), array([-23.03258091,  62.6242536 ]), array([-23.03258091,  62.6242536 ]), array([-23.13258091,  62.6242536 ]), array([-23.23258091,  62.6242536 ]), array([-23.23258091,  62.6242536 ]), array([-23.33258091,  62.6242536 ]), array([-23.33258091,  62.5242536 ]), array([-23.43258091,  62.5242536 ]), array([-23.53258091,  62.5242536 ]), array([-23.53258091,  62.5242536 ]), array([-23.63258091,  62.5242536 ]), array([-23.73258091,  62.5242536 ]), array([-23.73258091,  62.5242536 ]), array([-23.83258091,  62.5242536 ]), array([-23.83258091,  62.5242536 ]), array([-23.93258091,  62.5242536 ]), array([-24.03258091,  62.5242536 ]), array([-24.03258091,  62.6242536 ]), array([-24.13258091,  62.6242536 ]), array([-24.23258091,  62.6242536 ]), array([-24.23258091,  62.7242536 ]), array([-24.33258091,  62.7242536 ]), array([-24.33258091,  62.7242536 ]), array([-24.43258091,  62.7242536 ]), array([-24.53258091,  62.7242536 ]), array([-24.53258091,  62.8242536 ]), array([-24.63258091,  62.8242536 ]), array([-24.73258091,  62.8242536 ]), array([-24.73258091,  62.9242536 ]), array([-24.83258091,  62.9242536 ]), array([-24.83258091,  62.9242536 ]), array([-24.93258091,  62.9242536 ]), array([-25.03258091,  62.9242536 ]), array([-25.03258091,  63.0242536 ]), array([-25.13258091,  63.0242536 ]), array([-25.23258091,  63.0242536 ]), array([-25.23258091,  63.1242536 ]), array([-25.33258091,  63.1242536 ]), array([-25.33258091,  63.1242536 ]), array([-25.43258091,  63.1242536 ]), array([-25.53258091,  63.1242536 ]), array([-25.53258091,  63.2242536 ]), array([-25.63258091,  63.2242536 ]), array([-25.73258091,  63.2242536 ]), array([-25.73258091,  63.3242536 ]), array([-25.83258091,  63.3242536 ]), array([-25.93258091,  63.3242536 ]), array([-25.93258091,  63.4242536 ]), array([-26.03258091,  63.4242536 ]), array([-26.03258091,  63.5242536 ]), array([-26.13258091,  63.5242536 ]), array([-26.23258091,  63.5242536 ]), array([-26.23258091,  63.6242536 ]), array([-26.33258091,  63.6242536 ]), array([-26.43258091,  63.6242536 ]), array([-26.43258091,  63.7242536 ]), array([-26.53258091,  63.7242536 ]), array([-26.53258091,  63.8242536 ]), array([-26.63258091,  63.8242536 ]), array([-26.63258091,  63.9242536 ]), array([-26.73258091,  63.9242536 ]), array([-26.73258091,  64.0242536 ]), array([-26.83258091,  64.0242536 ]), array([-26.83258091,  64.1242536 ]), array([-26.93258091,  64.1242536 ]), array([-26.93258091,  64.2242536 ]), array([-27.03258091,  64.2242536 ]), array([-27.03258091,  64.3242536 ]), array([-27.13258091,  64.3242536 ]), array([-27.13258091,  64.4242536 ]), array([-27.13258091,  64.5242536 ]), array([-27.13258091,  64.5242536 ]), array([-27.13258091,  64.6242536 ]), array([-27.13258091,  64.6242536 ]), array([-27.13258091,  64.7242536 ]), array([-27.13258091,  64.8242536 ]), array([-27.13258091,  64.8242536 ]), array([-27.13258091,  64.9242536 ]), array([-27.13258091,  64.9242536 ])]",
```

Comment:
is this the same as the "bounding_polygon"?

Reply:
Yes. As Marta said in her previous comment, not all of the information in this file is used, but this is the information that is stored when the inversion has finished.